### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by kainino0x**
@kangz

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# GPU for the Web
+
+This repository is being used for work in the [W3C GPU for the Web Community
```
Comment (Kangz) [inline CONTRIBUTING.md]: Where did you get this template?

File: package.json [hunk 2]
```diff
@@ -10,9 +10,9 @@
     "type": "git",
     "url": "git+https://github.com/kainino0x/cts-experiment.git"
   },
-  "author": "Kai Ninomiya <kainino@chromium.org>",
+  "author": "WebGPU CTS Contributors",
   "private": true,
-  "license": "W3C 3-clause BSD License",
```
Comment (Kangz) [inline package.json]: Why not W3C 3-clause BSD?

File: CONTRIBUTING.md (hunk 1 above)
Comment (kainino0x) [reply CONTRIBUTING.md]: gpuweb/admin

File: package.json (hunk 2 above)
Comment (kainino0x) [reply package.json]: `BSD-3-Clause` is a standard (SPDX) string, so using it here allows machine understanding of the field.

AFAICT (from gpuweb/admin), the license that was agreed on for gpuweb is actually the vanilla BSD-3-Clause, not the W3C 3-clause BSD License. I checked that the LICENSE in gpuweb/admin is the SPDX BSD-3-Clause exactly.

**Pull request merged**